iree/compiler/Dialect/Flow/IR/FlowInterfaces.td

```diff
@@ -80,6 +80,14 @@ def FLOW_ClosureOpInterface : OpInterface<"ClosureOpInterface"> {
       /*args=*/(ins "ArrayRef<unsigned>":$excludedOperandIndices,
                     "ArrayRef<unsigned>":$excludedResultIndices)
     >,
+    InterfaceMethod<
+      /*desc=*/[{
+        Returns true if the output is also read within the region.
```

Comment:
It can be a bit confusing given we have both op results and region arguments representing those results. I'd suggest to say something like: "Returns true if the block argument matching the result at `resultIndex` is read within the region".

Reply:
Thats an implementation detail which is outside of the interface. For example `ExStream..` op does not have an argument for the output